File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/group-risk-employer-premium-rate--model.ts
import {PolicyPremiumRateDetailModel} from "./policy-premium-rate-detail-model";

export class GroupRiskEmployerPremiumRateModel {
  public employerRolePlayerId: number;
  public policyPremiumRateDetailModels: PolicyPremiumRateDetailModel[];
  public groupRiskNotes: number[];

  constructor() {
    this.policyPremiumRateDetailModels = [];
    this.groupRiskNotes = [];
  }
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Policy/GroupRisk/GroupRiskEmployerPremiumRateModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RMA.Service.ClientCare.Contracts.Entities.Policy.GroupRisk
{
    public class GroupRiskEmployerPremiumRateModel
    {
        public int EmployerRolePlayerId { get; set; }
        public List<PolicyPremiumRateDetailModel> PolicyPremiumRateDetailModels { get; set; }
        public GroupRiskEmployerPremiumRateModel()
        {
            this.PolicyPremiumRateDetailModels = new List<PolicyPremiumRateDetailModel>();
        }
    }
}
